spec/system/casa_cases/edit_spec.rb

```diff
@@ -2,6 +2,29 @@
 require "stringio"
 
 RSpec.describe "casa_cases/edit", :disable_bullet, type: :system do
+  shared_examples_for "shows past court dates links" do
+    let(:past_court_date_with_details) do
+      create(:past_court_date, :with_court_details, casa_case: casa_case, date: Time.zone.yesterday)
+    end
+
+    let(:past_court_date_without_details) do
+      create(:past_court_date, casa_case: casa_case, date: Time.zone.today)
+    end
+
+    let!(:formatted_date_with_details) { I18n.l(past_court_date_with_details.date, format: :full, default: nil) }
+    let!(:formatted_date_without_details) { I18n.l(past_court_date_without_details.date, format: :full, default: nil) }
```

Comment:
I would rather spell this out than reference the yaml- tests don't need to be as DRY as prod code. tests Should not test prod code by running other prod code